### DISCUSSION TITLE
Can't able to change wayber default font_size.

### DISCUSSION THREAD

**Discussion by b1kr3m** [prasanthrangan/hyprdots#812] [Q&A] [answered]
actually I have less  knowledge in css and jsonc. Whenever i tried the to change font_size from style.css it changed, but after toggle to another menu bar it become same as usual. So how i can change my font_size permanent.

![16px font size](https://github.com/prasanthrangan/hyprdots/assets/97847882/d27cade2-d35a-4091-af6b-c8cede05b102)

![font_size = 10](https://github.com/prasanthrangan/hyprdots/assets/97847882/9b570f78-0500-4ed7-b7a9-d2155ebda965)

please guide me how can i do that for every waybar module.

**Comment by kRHYME7** [comment, marked as answer]
https://github.com/prasanthrangan/hyprdots/wiki/Theming

Hello, I hope sending just a link won't be rude. I want to utilize the wiki. BTW check the waybar part.

**Reply by b1kr3m** [reply]
my apology.. i understood my problem. thanks for guide me.
